README.md

```diff
@@ -48,15 +48,7 @@ Samson will:
 
 ### Contributing
 
-Improvements are always welcome. Please follow these steps to contribute
-
-1. Submit a Pull Request with a detailed explanation of changes and screenshots (if UI is changing)
-1. Receive a :+1: from a core team member
-1. Core team will merge your changes
-
-### Team
-
-Core team is [@henders](https://github.com/henders), [@grosser](https://github.com/grosser), [@jonmoter](https://github.com/jonmoter), [@irwaters](https://github.com/irwaters).
```

Comment:
what about having it as a team? like `@zendesk/samson`

Reply:
`@zendesk/samson` is a team

Reply:
not sure those outside the org can mention a team..

Reply:
yeah true ... we got code-owners for that, but won't work for issues :/

Reply:
removed the team mention since that won't work for public contributors anyway and the code-owners will already send the PR review request